projects/pluto/src/main/java/nl/fairspace/pluto/web/IconsResource.java

```diff
@@ -0,0 +1,55 @@
+package nl.fairspace.pluto.web;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.io.IOUtils;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import nl.fairspace.pluto.config.dto.PlutoConfig;
+
+import static nl.fairspace.pluto.config.Urls.ICONS_PATH;
+
+@RestController
+@Slf4j
+@AllArgsConstructor
```

Comment:
Resource is a singleton and unmodifiable. This is correct to define fields final, `@RequiredArgsConstructor` can be used instead

Reply:
Does it matter in this case if I use RequiredArgsConstructor vs AllArgsConstructor?

Reply:
not critical,  the only advantage is that it will be safer to add new dependencies